src/doc_emit/html_main.re

```diff
@@ -203,7 +207,9 @@ and show_value = (~options as {resolve, _} as options, value) => {
         ...{
              fs
              |> List.map(((field, value)) =>
-                  <tr>
+                  <tr class_={
+                    value.deprecated != None ? "definition-deprecated" : ""
```

Comment:
`Option.is_some(value.deprecated)` probably. Doing equality on ADTs rather than pattern matching is always a bit of a bad look.

In fact, thanks to the magic of Squid actually reading the docs, this can be replaced with:
```reason
<tr class_=?{Option.map((_) => "definition-deprecated", value.deprecated)}>
```

You'll need to merge in/rebase onto master for this to work - was missing some functionality on my end.

Reply:
Thanks for adding that!